### DISCUSSION TITLE
Printing ASA ADC error

### DISCUSSION THREAD

**Discussion by Trilis29** [bassamanator/Sovol-SV06-firmware#59] [Q&A]
mcu shutdown ADC out of range. Is this printer related?

Hotend 260 
Bed 105

**Comment by bassamanator** [comment]
It's possible that since your bed is likely not PID tuned to > 100C, it's going all over the place when it gets close to a 100C, likely going over 110C, which is set as the max. 

My thinking is that PID tuning your bed to 105C will likely solve your issue. I guess you could also set `[heater_bed]` `max_temp` to 115, but I recommend a PID tuning.

**Comment by Trilis29** [comment]
Set max_temp to 111 and pid tune to 110... Problem solved printing ASA without enclosure 🤘🤘🤘

**Reply by bassamanator** [reply]
@Trilis29 Great!

Please ⭐star⭐ the repo if you found this config useful. Happy printing!
